src/__tests__/acceptance/test-helper.ts

```diff
@@ -0,0 +1,32 @@
+import {FluffyOctoEngineApplication} from '../..';
+import {
+  createRestAppClient,
```

Comment:
:robot: OpenAI

The `setupApplication` function is missing a return type. It should be `Promise<AppWithClient>`.
```suggestion
export async function setupApplication(): Promise<AppWithClient> {
```

<!-- This is an auto-generated comment by OpenAI -->